pkg/asset/manifests/openshift.go

```diff
@@ -270,6 +270,15 @@ func (o *Openshift) Generate(ctx context.Context, dependencies asset.Parents) er
 			ProvisioningOSDownloadURL: string(*rhcosImage),
 		}
 		assetData["99_baremetal-provisioning-config.yaml"] = applyTemplateData(baremetalConfig.Files()[0].Data, bmTemplateData)
+
+		// Create cert to enable TLS for virtual media
+		if installConfig.Config.BareMetal.DisableVirtualMediaTLS {
+			cert, err := generateTlsCertificate(installConfig.Config.Platform.Baremetal.BootstrapProvisioningIP)
```

Comment:
You can generate the certificate unconditionally. The final decision will need to be done inside the bootstrap VM since at that point the final certificates will be available.

Reply:
> Which part has to be done inside the bootstrap?
This whole bug is for the bootstrap VM, if that's what you are asking.